recipes/extra-cmake-modules/all/test_package/CMakeLists.txt

```
@@ -0,0 +1,21 @@
+cmake_minimum_required(VERSION 3.0)
```

The hooks require at least 3.1 for the test_package
```suggestion
cmake_minimum_required(VERSION 3.1)
```
See https://github.com/conan-io/hooks/blob/e62ba99e29b963088f73611693ffd3144b62e45e/hooks/conan-center.py#L495